### PR TITLE
Docs/translate electronics

### PR DESCRIPTION
**Pull request by jgueldenstein**
## Necessary checks
- [x] Update package version
- ~~[ ] Run linters~~
- [x] Run `catkin build`
- [x] Write documentation
- ~~[ ] Test on your machine~~
- ~~[ ] Test on the robot~~

### CODE REVIEW

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+==========
+Elektronik
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]:
```suggestion
Electronics
```
When applying this, the "=" above and below also need to be expanded by one "=" each.

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+==========
+
+Powerboard
+==========
+
+The Powerboard allows to connect either and external power supply or a battery. It has a switch to turn on and off the power to the servos.
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]:
```suggestion
The Powerboard allows to connect either an external power supply or a battery. It has a switch to turn on and off the power to the servos.
```

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Documentation about the Hardware is available at [https://github.com/Rhoban/DXLBoard](https://github.com/Rhoban/DXLBoard).
+We use a custom firmware to support Dynamixel Protocol 2.0 which can be found here: [https://github.com/bit-bots/DXLBoard/tree/protocol2_single_bus](https://github.com/bit-bots/DXLBoard/tree/protocol2_single_bus)
+
+Wolfgang Core
+=============
+
+Wolfgang Core (COntrolling and Regulating Electronics) is currently in the testing phase.
+
+* 4 buses with up to 10 MBaud (highest for Dynamixel MX and X servos is 4 MBaud)
+* power swiching with switch and software
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]:
```suggestion
* power switching with switch and software
```

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+* voltage regulation (9V 1A, 5V 5A)
+* voltage sensing
+* current sensing
+* 3 RGB status leds
+
+
+Bit Foot
+========
+
+Der Bit Foot is a board to read 4 load cells on the foot of the robot and communicate over the dynamixel bus.
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]:
```suggestion
The Bit Foot is a board to read 4 load cells on the foot of the robot and communicate over the dynamixel bus.
```

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
+==========
+Elektronik
+==========
+
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]: it seems like this empty line shouldn't be here?

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+The footprint of the mosfet is currently wrong, but since the Wolfgang Core board integrates the functionality of this board, it will not be changed.
+
+
+DXLBoard
+========
+
+The DXLBoard is developed by Rhoban from Bordeaux. It communicates with the Servo bus over RS485 or TTL and the NUC via USB.
+
+Documentation about the Hardware is available here_.
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]: Does this link work?
Github renders it as underlined text, but it is not clickable.

File: bitbots_docs/docs/manual/hardware/electronics.rst [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+* current sensing
+* 3 RGB status leds
+
+
+Bit Foot
+========
+
+The Bit Foot is a board to read 4 load cells on the foot of the robot and communicate over the dynamixel bus.
+
+* Schaltplan :download:`pdf <elektronik/bitfoot.pdf>`
```
Comment (johagge) [inline bitbots_docs/docs/manual/hardware/electronics.rst]:
```suggestion
* Schematics :download:`pdf <elektronik/bitfoot.pdf>`
```

File: bitbots_docs/docs/manual/hardware/electronics.rst (hunk 6 above)
Comment (jgueldenstein) [reply bitbots_docs/docs/manual/hardware/electronics.rst]: This style of link worked in the previous version of the document,

**Pull request merged**